src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -434,6 +411,41 @@ impl<S: PkAndRowStream + Unpin> TableIter for S {
             .transpose()
             .map(|r| r.map(|(_pk, row)| row))
     }
+
+    async fn collect_data_chunk(
```

Comment:
Why do we need this? 👀

Reply:
I believe this was generated by git merge (not my code😅)